django/views/templates/technical_500.html

```diff
@@ -137,7 +138,7 @@ <h1>{% if exception_type %}{{ exception_type }}{% else %}Report{% endif %}
 {% if lastframe %}
     <tr>
       <th>Exception Location:</th>
-      <td>{{ lastframe.filename }} in {{ lastframe.function }}, line {{ lastframe.lineno }}</td>
+      <td><span class="fpos">{{ lastframe.filename }}:{{ lastframe.lineno }}</span> (in {{ lastframe.function }})</td>
```

Comment:
For me (a person that doesn't use editor which handle this format) it is less readable, TBH. I also think it is less readable for most of users. I would leave the previous format and add `lineno` after the filename, e.g.
```HTML
<td>
    <span class="fpos">{{ lastframe.filename }}:{{ lastframe.lineno }}</span>
    in {{ lastframe.function }}, line {{ lastframe.lineno }}
</td>
```

Reply:
@felixxm 
Thanks for the feedback.
In your example the line number will appear twice though then.
This would be ok with me, but might be confusing.
For me it would be helpful already if the filename would be more easily selectable through double-clicking it.

Reply:
/cc @carltongibson (via https://code.djangoproject.com/ticket/30779)

Reply:
If that's OK with you, we can stay with:
```html
<td>
    <span class="fpos">{{ lastframe.filename }}</span> in {{ lastframe.function }}, line {{ lastframe.lineno }}
</td>
```

Reply:
Ok, amended.
Will adjust tests once logs are there again.

Reply:
Sorry, for bike-shedding: what about:

> path/to/file.py, line 42, in func

This would match the order with tracebacks.

Apart from that I can imagine user's being annoyed by not being able to only select part of the filename anymore with using `user-select: all;` there.

Reply:
> > path/to/file.py, line 42, in func

Looks good.